response.go

```diff
@@ -120,19 +120,22 @@ func (r *Response) RawBody() io.ReadCloser {
 }
 
 func (r *Response) fmtBodyString() string {
-	bodyStr := "***** NO CONTENT *****"
 	if r.body != nil {
+		if len(r.body) > 1000000 {
+			return "***** RESPONSE TOO LARGE *****"
+		}
 		ct := r.Header().Get(hdrContentTypeKey)
 		if IsJSONType(ct) {
 			out := acquireBuffer()
 			defer releaseBuffer(out)
-			if err := json.Indent(out, r.body, "", "   "); err == nil {
-				bodyStr = string(out.Bytes())
+			err := json.Indent(out, r.body, "", "   ")
```

Comment:
This could be better and concise.

```go
if err := json.Indent(out, r.body, "", "   "); err == nil {
    return out.String()
}
```

Reply:
That results in a warning from golint - `if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary) (golint)`.

Reply:
Actually remove this line `return "unexpected error: " + err.Error()` with my above code snippet then golint warning goaway. Because if any errors in formatting debug log, return the original body value which is `r.String()`.